src/gui/guimain.cpp

```diff
@@ -80,8 +80,8 @@ void MsgCircleBuffer_cb(const char *txt) {
 }
 
 void Warning_cb(WarningType type) {
-    static constexpr const char *HotendFanErrorMsg = N_("I am dummy HotendFanErrorMsg, I need to be replaced with something else ... ");
-    static constexpr const char *PrintFanErrorMsg = N_("I am dummy PrintFanError, I need to be replaced with something else ... ");
+    static constexpr const char *HotendFanErrorMsg = N_("Err: EXTR. FAN ERROR");
+    static constexpr const char *PrintFanErrorMsg = N_("Err: PRINT FAN ERROR");
```

Comment:
The "error" world is there two times.

What about something more user friendly, like:
```C++
static constexpr const char *HotendFanErrorMsg = N_("Extruder fan error");
static constexpr const char *PrintFanErrorMsg = N_("Print fan error");
```

Reply:
Solved